Generals/Code/GameEngine/Source/Common/INI/INI.cpp

```diff
@@ -1528,7 +1528,10 @@ void INI::initFromINIMulti( void *what, const MultiIniFieldParse& parseTableList
 				{
 					DEBUG_ASSERTCRASH( 0, ("[LINE: %d - FILE: '%s'] Unknown field '%s' in block '%s'",
 														 INI::getLineNum(), INI::getFilename().str(), field, m_curBlockStart) );
+#if !defined(RTS_DEBUG)
+					// Just ignore unknown fields in debug builds
```

Comment:
Perhaps stop throwing in Release too? Or does a superfluous field indeed warrant crashing for the end user?